rudder-web/src/main/webapp/javascript/rudder/rudder.js

```diff
@@ -606,4 +589,74 @@ $(document).ready(function() {
   createTooltip();
 });
 
+function createTextAgentPolicyMode(isNode, currentPolicyMode, explanation){
+  var policyMode = currentPolicyMode.toLowerCase();
+  var nodeOrDirective = isNode ? "node" : "directive";
+  var labelType = "label-"+policyMode;
+  var policyName = policyMode == "verify" ? "audit" : policyMode
+  var span = "<span class='label-text " + labelType + " glyphicon glyphicon-question-sign' data-toggle='tooltip' data-placement='top' data-html='true' title=''></span>"
+  var badge = $(span).get(0);
+  var tooltip = "" +
```

Comment:
I think the tooltip can be put in an external function ( like : 'policyModetooltip (kind, policyName, explanation)'